merkledag/merkledag.go

```diff
@@ -175,7 +175,15 @@ func FetchGraph(ctx context.Context, root *cid.Cid, serv DAGService) error {
 	set := cid.NewSet()
 	visit := func(c *cid.Cid) bool {
 		if set.Visit(c) {
-			v.Increment()
+			cnt := 0
+			node, err := serv.Get(ctx, c)
+			if err == nil {
+				stat, err := node.Stat()
+				if err == nil {
+					cnt = stat.LinksSize + stat.DataSize
```

Comment:
I'm not entirely sure if this is correct. I tried using `stat.BlockSize` but that did not seem to include the DataSize measurement.